src/components/ResourceList.tsx

```diff
@@ -18,10 +18,15 @@ interface ResourceList {
             alt: string
         }
     }[]
+    parentVertPadding?: boolean
 }
 
-export const ResourceList: FunctionComponent<ResourceList> = ({ items, title = 'Related Resources' }) => (
-    <ContentSection background="white">
+export const ResourceList: FunctionComponent<ResourceList> = ({
+    items,
+    title = 'Related Resources',
+    parentVertPadding = true,
+}) => (
+    <ContentSection background="white" parentVertPadding={parentVertPadding}>
```

Comment:
AbrevsCanSomtmzBeHrdToReadSoLetsUseFullWrdz

```suggestion
export const ResourceList: FunctionComponent<ResourceList> = ({
    items,
    title = 'Related Resources',
    parentVerticalPadding = true,
}) => (
    <ContentSection background="white" parentVertPadding={parentVerticalPadding}>
```